### DISCUSSION TITLE
Unswap CAKE from pancake swap V1 with Trust Wallet V2 (I cannot unswap my cake from 2 years ago)

### DISCUSSION THREAD

**Discussion by Jaben24** [WalletConnect/walletconnect-monorepo#5518] [Developer Support]
Hello dear all,

I stacked a big amount of cake 2 years ago, I cannot unswap my cake from pancakeswap v1 syrup because they migrate to V2 as well and because once I want to connect the Trust Wallet to the pancakeswap v1 syrup staked cake (I get an error that says: DApp uses unsupported version (1) of WalletConnect)  Can somebody help me with this issue please!?! Thanks in adance

**Comment by arein** [comment]
@Jaben24 this is an issue for Pancakeswap not us so closing

But you can potentially export your keys into a browser extension wallet and then use that on the old pancakceswap version
